flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/cleanup/DispatcherResourceCleanerFactory.java

```diff
@@ -108,6 +108,7 @@ public ResourceCleaner createGlobalResourceCleaner(
                 .withRegularCleanup(jobGraphWriter)
                 .withRegularCleanup(blobServer)
                 .withRegularCleanup(highAvailabilityServices)
+                .withRegularCleanup(jobManagerMetricGroup)
```

Comment:
Off-topic: Why are there no javadocs explaining the difference between regular and prioritized cleanups?

Reply:
OT: typo in the `createExponentialRetryStategy` method below